pkgs/data/icons/numix-icon-theme-circle/default.nix

```diff
@@ -1,7 +1,7 @@
 { stdenv, fetchFromGitHub, gtk3, numix-icon-theme }:
 
 stdenv.mkDerivation rec {
-  version = "18.09.19";
+  version = "19.01.24";
 
   package-name = "numix-icon-theme-circle";
```

Comment:
Probably should switch to using `pname`.

Reply:
Done.